SMLHelper/SMLHelper.csproj

```diff
@@ -47,17 +47,25 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="0Harmony">
-      <HintPath>..\Dependencies\0Harmony-1.2.0.1.dll</HintPath>
+      <HintPath>..\..\DependenciesBZ\0Harmony-1.2.0.1.dll</HintPath>
+      <Private>False</Private>
     </Reference>
     <Reference Include="Assembly-CSharp-firstpass_publicized">
-      <HintPath>..\Dependencies\Assembly-CSharp-firstpass_publicized.dll</HintPath>
+      <HintPath>..\..\DependenciesBZ\Assembly-CSharp-firstpass_publicized.dll</HintPath>
+      <Private>False</Private>
```

Comment:
**TODO** Handle the version specific references based on the build configuration